doc/Command-Reference.md

```diff
@@ -1058,6 +1058,289 @@ This command displays information for all the interfaces for the transceiver req
+          Tx bias high alarm flag on lane 4: False
+          Tx bias high alarm flag on lane 5: False
+          Tx bias high alarm flag on lane 6: False
+          Tx bias high alarm flag on lane 7: False
+          Tx bias high alarm flag on lane 8: False
+          Tx bias low alarm flag on lane 1: False
+          Tx bias low alarm flag on lane 2: False
+          Tx bias low alarm flag on lane 3: False
+          Tx bias low alarm flag on lane 4: False
+          Tx bias low alarm flag on lane 5: False
+          Tx bias low alarm flag on lane 6: False
+          Tx bias low alarm flag on lane 7: False
+          Tx bias low alarm flag on lane 8: False
+          Tx bias high warning flag on lane 1: False
+          Tx bias high warning flag on lane 2: False
+          Tx bias high warning flag on lane 3: False
+          Tx bias high warning flag on lane 4: False
+          Tx bias high warning flag on lane 5: False
+          Tx bias high warning flag on lane 6: False
+          Tx bias high warning flag on lane 7: False
+          Tx bias high warning flag on lane 8: False
+          Tx bias low warning flag on lane 1: False
+          Tx bias low warning flag on lane 2: False
+          Tx bias low warning flag on lane 3: False
+          Tx bias low warning flag on lane 4: False
+          Tx bias low warning flag on lane 5: False
+          Tx bias low warning flag on lane 6: False
+          Tx bias low warning flag on lane 7: False
+          Tx bias low warning flag on lane 8: False
+          Laser temperature high alarm flag: False
+          Laser temperature low alarm flag: False
+          Laser temperature high warning flag: False
+          Laser temperature low warning flag: False
+          Prefec ber high alarm flag: False
+          Prefec ber low alarm flag: False
+          Prefec ber high warning flag: False
+          Prefec ber low warning flag: False
+          Postfec ber high alarm flag: False
+          Postfec ber low alarm flag: False
+          Postfec ber high warning flag: False
+          Postfec ber low warning flag: False
+          Bias xi high alarm flag: False
+          Bias xi low alarm flag: False
+          Bias xi high warning flag: False
+          Bias xi low warning flag: False
+          Bias xq high alarm flag: False
+          Bias xq low alarm flag: False
+          Bias xq high warning flag: False
+          Bias xq low warning flag: False
+          Bias xp high alarm flag: False
+          Bias xp low alarm flag: False
+          Bias xp high warning flag: False
+          Bias xp low warning flag: False
+          Bias yi high alarm flag: False
+          Bias yi low alarm flag: False
+          Bias yi high warning flag: False
+          Bias yi low warning flag: False
+          Bias yq high alarm flag: False
+          Bias yq low alarm flag: False
+          Bias yq high warning flag: False
+          Bias yq low warning flag: False
```

Comment:
@longhuan-cisco can we separate our the output such a way the status output for 400G ZR is not displayed for CMIS non-ZR ? Similar to DOM https://github.com/sonic-net/SONiC/pull/1306/files

Reply:
This is already guaranteed. since CLI will only show based on DB, and non-ZR optics don't have ZR related fields in DB.

CMIS non-ZR sample output:
```
Ethernet264:                                                                                                                                                                                                                                                 
        Current module state: ModuleReady                                                                                                                                                                                                                    
        Reason of entering the module fault state: No Fault detected                                                                                                                                                                                         
        Datapath firmware fault: False                                                                                                                                                                                                                       
        Module firmware fault: False                                                                                                                                                                                                                         
        Module state changed: False                                                                                                                                                                                                                          
        Data path state indicator on host lane 1: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 2: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 3: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 4: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 5: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 6: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 7: DataPathActivated                                                                                                                                                                                          
        Data path state indicator on host lane 8: DataPathActivated                                                                                                                                                                                          
        Tx output status on media lane 1: False                                                                                                                                                                                                              
        Tx output status on media lane 2: False                                                                                                                                                                                                              
        Tx output status on media lane 3: False                                                                                                                                                                                                              
        Tx output status on media lane 4: False                                                                                                                                                                                                              
        Tx output status on media lane 5: False                                                                                                                                                                                                              
        Tx output status on media lane 6: False                                                                                                                                                                                                              
        Tx output status on media lane 7: False                                                                                                                                                                                                              
        Tx output status on media lane 8: False                                                                                                                                                                                                              
        Rx output status on host lane 1: False                                                                                                                                                                                                               
        Rx output status on host lane 2: False                                                                                                                                                                                                               
        Rx output status on host lane 3: False                                                                                                                                                                                                               
        Rx output status on host lane 4: False                                                                                                                                                                                                               
        Rx output status on host lane 5: False                                                                                                                                                                                                               
        Rx output status on host lane 6: False                                                                                                                                                                                                               
        Rx output status on host lane 7: False                                                                                                                                                                                                               
        Rx output status on host lane 8: False
        Disabled TX channels: 0
        TX disable status on lane 1: False
        TX disable status on lane 2: False
        TX disable status on lane 3: False
        TX disable status on lane 4: False
        TX disable status on lane 5: False
        TX disable status on lane 6: False
        TX disable status on lane 7: False
        TX disable status on lane 8: False                                                                                                                                                                                                               
        Tx fault flag on media lane 1: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 2: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 3: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 4: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 5: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 6: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 7: N/A                                                                                                                                                                                                                   
        Tx fault flag on media lane 8: N/A                                                                                                                                                                                                                   
        Tx loss of signal flag on host lane 1: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 2: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 3: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 4: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 5: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 6: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 7: False                                                                                                                                                                                                         
        Tx loss of signal flag on host lane 8: False                                                                                                                                                                                                         
        Tx clock and data recovery loss of lock on host lane 1: False                                                                                                                                                                                        
        Tx clock and data recovery loss of lock on host lane 2: False                                                                                                                                                                                        
        Tx clock and data recovery loss of lock on host lane 3: False                                                                                                                                                                                        
        Tx clock and data recovery loss of lock on host lane 4: False    
        Tx clock and data recovery loss of lock on host lane 5: False                                                                                                                                                                                        
        Tx clock and data recovery loss of lock on host lane 6: False                                                                                                                                                                                        
        Tx clock and data recovery loss of lock on host lane 7: False                                                                                                                                                                                        
        Tx clock and data recovery loss of lock on host lane 8: False                                                                                                                                                                                        
        Rx loss of signal flag on media lane 1: False                                                                                                                                                                                                        
        Rx loss of signal flag on media lane 2: False                                                                                                                                                                                                        
        Rx loss of signal flag on media lane 3: False
        Rx loss of signal flag on media lane 4: False
        Rx loss of signal flag on media lane 5: False
        Rx loss of signal flag on media lane 6: False
        Rx loss of signal flag on media lane 7: False
        Rx loss of signal flag on media lane 8: False
        Rx clock and data recovery loss of lock on media lane 1: False
        Rx clock and data recovery loss of lock on media lane 2: False
        Rx clock and data recovery loss of lock on media lane 3: False
        Rx clock and data recovery loss of lock on media lane 4: False
        Rx clock and data recovery loss of lock on media lane 5: False
        Rx clock and data recovery loss of lock on media lane 6: False
        Rx clock and data recovery loss of lock on media lane 7: False
        Rx clock and data recovery loss of lock on media lane 8: False
        Configuration status for the data path of host line 1: ConfigSuccess
        Configuration status for the data path of host line 2: ConfigSuccess
        Configuration status for the data path of host line 3: ConfigSuccess
        Configuration status for the data path of host line 4: ConfigSuccess
        Configuration status for the data path of host line 5: ConfigSuccess
        Configuration status for the data path of host line 6: ConfigSuccess
        Configuration status for the data path of host line 7: ConfigSuccess
        Configuration status for the data path of host line 8: ConfigSuccess
        Data path configuration updated on host lane 1: False
        Data path configuration updated on host lane 2: False
        Data path configuration updated on host lane 3: False
        Data path configuration updated on host lane 4: False
        Data path configuration updated on host lane 5: False
        Data path configuration updated on host lane 6: False
        Data path configuration updated on host lane 7: False
        Data path configuration updated on host lane 8: False
        Temperature high alarm flag: False
        Temperature high warning flag: False
        Temperature low warning flag: False
        Temperature low alarm flag: False
        Vcc high alarm flag: False
        Vcc high warning flag: False
        Vcc low warning flag: False
        Vcc low alarm flag: False
        Tx power high alarm flag on lane 1: False
        Tx power high alarm flag on lane 2: False
        Tx power high alarm flag on lane 3: False
        Tx power high alarm flag on lane 4: False
        Tx power high alarm flag on lane 5: False
        Tx power high alarm flag on lane 6: False
        Tx power high alarm flag on lane 7: False
        Tx power high alarm flag on lane 8: False
        Tx power high warning flag on lane 1: False
        Tx power high warning flag on lane 2: False
        Tx power high warning flag on lane 3: False
        Tx power high warning flag on lane 4: False                                                                                                                                                                                                          
        Tx power high warning flag on lane 5: False                                                                                                                                                                                                          
        Tx power high warning flag on lane 6: False                                                                                                                                                                                                          
        Tx power high warning flag on lane 7: False                                                                                                                                                                                                          
        Tx power high warning flag on lane 8: False                                                                                                                                                                                                          
        Tx power low alarm flag on lane 1: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 2: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 3: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 4: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 5: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 6: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 7: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 8: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 1: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 2: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 3: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 4: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 5: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 6: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 7: False                                                                                                                                                                                                             
        Tx power low alarm flag on lane 8: False                                                                                                                                                                                                             
        Rx power high alarm flag on lane 1: False                                                                                                                                                                                                            
        Rx power high alarm flag on lane 2: False                                                                                                                                                                                                            
        Rx power high alarm flag on lane 3: False                                                                                                                                                                                                            
        Rx power high alarm flag on lane 4: False                                                                                                                                                                                                            
        Rx power high alarm flag on lane 5: False
        Rx power high alarm flag on lane 6: False
        Rx power high alarm flag on lane 7: False
        Rx power high alarm flag on lane 8: False
        Rx power high warning flag on lane 1: False
        Rx power high warning flag on lane 2: False                                                                                                                                                                                               
        Rx power high warning flag on lane 3: False                                                                                                                                                                                                          
        Rx power high warning flag on lane 4: False                                                                                                                                                                                                          
        Rx power high warning flag on lane 5: False                                                                                                                                                                                                          
        Rx power high warning flag on lane 6: False                                                                                                                                                                                                          
        Rx power high warning flag on lane 7: False                                                                                                                                                                                                          
        Rx power high warning flag on lane 8: False                                                                                                                                                                                                          
        Rx power low warning flag on lane 1: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 2: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 3: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 4: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 5: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 6: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 7: False                                                                                                                                                                                                           
        Rx power low warning flag on lane 8: False                                                                                                                                                                                                           
        Rx power low alarm flag on lane 1: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 2: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 3: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 4: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 5: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 6: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 7: False                                                                                                                                                                                                             
        Rx power low alarm flag on lane 8: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 1: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 2: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 3: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 4: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 5: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 6: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 7: False                                                                                                                                                                                                             
        Tx bias high alarm flag on lane 8: False                                                                                                                                                                                                             
        Tx bias high warning flag on lane 1: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 2: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 3: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 4: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 5: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 6: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 7: False                                                                                                                                                                                                           
        Tx bias high warning flag on lane 8: False                                                                                                                                                                                                           
        Tx bias low warning flag on lane 1: False                                                                                                                                                                                                            
        Tx bias low warning flag on lane 2: False
        Tx bias low warning flag on lane 3: False
        Tx bias low warning flag on lane 4: False
        Tx bias low warning flag on lane 5: False
        Tx bias low warning flag on lane 6: False
        Tx bias low warning flag on lane 7: False
        Tx bias low warning flag on lane 8: False
        Tx bias low alarm flag on lane 1: False
        Tx bias low alarm flag on lane 2: False
        Tx bias low alarm flag on lane 3: False
        Tx bias low alarm flag on lane 4: False
        Tx bias low alarm flag on lane 5: False
        Tx bias low alarm flag on lane 6: False
        Tx bias low alarm flag on lane 7: False
        Tx bias low alarm flag on lane 8: False
        Laser temperature high alarm flag: False
        Laser temperature high warning flag: False
        Laser temperature low warning flag: False
        Laser temperature low alarm flag: False
```

Reply:
Changed to use separate map for CMIS and C_CMIS: CMIS_STATUS_MAP and CCMIS_STATUS_MAP